src/components/posts/PostItem.js

```diff
@@ -1,13 +1,20 @@
+import { useState } from "react/cjs/react.development";
 import Post from "../UI/Post";
+import Backdrop from "../UI/Backdrop";
 import DeleteButton from "./buttons/DeleteButton";
+import EditModal from "./buttons/EditModal";
+import LikeButton from "./buttons/LikeButton";
 import classes from "./PostItem.module.css";
 
 function PostItem(props) {
-  function likeHandler() {
-    console.log(props.id);
-  }
+  const [showEdit, setShowEdit] = useState(false);
+  const [likeCount, setLikeCount] = useState(props.likes);
   function editHandler() {
-    console.log(props.id);
+    if (showEdit) {
```

Comment:
This can be done in a much easier way by not using an `if` statement

Reply:
yes